src/state/slices/marketDataSlice/selectors.ts

```diff
@@ -13,21 +13,15 @@ import type { ReduxState } from '@/state/reducer'
 import { createDeepEqualOutputSelector } from '@/state/selector-utils'
 import { selectAssetIdParamFromFilter } from '@/state/selectors'
 import type { PriceHistoryData } from '@/state/slices/marketDataSlice/types'
-import { selectSelectedCurrency } from '@/state/slices/preferencesSlice/selectors'
+import { preferences } from '@/state/slices/preferencesSlice/preferencesSlice'
 
-export const selectMarketDataIdsSortedByMarketCapUsd = (state: ReduxState) =>
-  state.marketData.crypto.ids
-const selectFiatMarketData = (state: ReduxState) => state.marketData.fiat.byId
-
-export const selectMarketDataUsd = ((state: ReduxState) => state.marketData.crypto.byId) as (
-  state: ReduxState,
-) => MarketDataById<AssetId>
+export const selectMarketDataUsd = marketData.selectors.selectMarketDataUsd
```

Comment:
nitpik - follow pattern of `selectors.` usage instead of re-export var

Reply:
369094bf63